.gitignore

```diff
@@ -0,0 +1,330 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
You probably just want to remove this, seems generated by VS more than likely.

Reply:
.gitignore files exist specifically to _prevent_ auto-generated junk from infiltrating Github repos. I use Visual Studio, so I have a Visual Studio .gitignore in my working fork and branch. This can and probably should be removed from any published release, of course.

Reply:
Well I said that because it's mainly for C#, and executable files and so on, there's nothing here for prophunt, or gLua